packages/@sfcc-bff/basketapi/src/api/schema/basketResolvers.ts

```diff
@@ -63,27 +70,32 @@ const getBasket = async (config, context) => {
         return basket;
     }
 
-    basket.getCartMessage = `Basket found with ID of ${basketId}`;
+    basket.getBasketMessage = `Basket found with ID of ${basketId}`;
 
     // Get Shipping Methods
+    if (!basket?.shipments?.length || !basket.shipments[0].shipmentId)
+        return { fault: { message: 'No available shipment methods!' } };
```

Comment:
TypeScript forces us to handle the case where the shipments array is empty or non existent.